Create new files to plot S6

Albertlong039 opened the pull request:
add new codes to plot spread time vs network size under different q

Albertlong039 committed:
Create new files to plot S6

add new codes to plot spread time vs network size under different q

Albertlong039 committed:
Update computing_spread_time_c_1_union_ER_vs_network_size.py

aminrahimian commented on computing_spread_time_c_1_union_ER_vs_network_size.py:
```diff
@@ -0,0 +1,129 @@
+
+size_of_dataset = 2
+
+NET_SIZE = 10
```
Can you check the NET_SIZE here, does seem consistent with Figure S6C

aminrahimian replied:
Please commit your slurm files as well.

aminrahimian commented on computing_spread_time_c_1_union_ER_vs_network_size.py:
```diff
@@ -0,0 +1,129 @@
+        spread_time_stds = []
+
+
+        for Q in fixed_prob_low:
```
where do you record the NET_SIZE in these pickle files as you run them for different NET_SIZE values? I don't see it in file names. You can also record them in the path (each NET_SIZE in a different folder).

Albertlong039 committed:
Update computing_spread_time_c_1_union_ER_vs_network_size.py

Albertlong039 committed:
add new slurm file and reorganize the file to generate the pickle files

add new slurm file and reorganize the compute file to S6 to generate the pickle files

Albertlong039 committed:
Update computing_spread_time_c_1_union_ER_vs_network_size.py

update a small bug of output

aminrahimian reviewed (commented):
The slurm file and the reorganization of pickle files in NET_SIZE folders look great to me.

aminrahimian reviewed (commented):
The slurm file and the reorganization of pickle files in NET_SIZE folders look great to me.

aminrahimian reviewed (approved):
c1 union ER vs network size results seem complete. Good job! @Albertlong039
